src/Symfony/Component/VarDumper/Tests/CliDumperTest.php

```diff
@@ -107,6 +107,92 @@ class: "Symfony\Component\VarDumper\Tests\CliDumperTest"
         );
     }
 
+    public function testGetWithCommaFlag()
+    {
+        require __DIR__.'/Fixtures/dumb-var.php';
```

Comment:
I suggest to use a smaller fixture - one that contains a multiline string - and to test also the `DUMP_TRAILING_COMMA` flag